.ddev/commands/web/openmage-install

```diff
@@ -123,8 +123,8 @@ else
   if [[ $INSTALL_SAMPLE_DATA =~ ^(yes|y) ]]; then
     TABLE_PREFIX=''
   else
-    read -r -p "Table prefix []: " TABLE_PREFIX
-    TABLE_PREFIX=${TABLE_PREFIX:-}
+    read -r -p "Table prefix [om_]: " TABLE_PREFIX
+    TABLE_PREFIX=${TABLE_PREFIX:-om_}
```

Comment:
What happens if I don't want a table prefix when I am not using the sample data? By default, 'om_' is added if I press Enter. The previous version was the correct one in my opinion.

Reply:
Good catch. Will update tomorrow.

imho ... table-prefix should be used by default to find errors in modules that dont care about possible set prefix.

Reply:
fixed